applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_adjoint_potential_flow_element.cpp

```diff
@@ -12,6 +12,7 @@
 //
 #include "compressible_potential_flow_application_variables.h"
 #include "incompressible_potential_flow_element.h"
+#include "compressible_potential_flow_element.h"
```

Comment:
Just wondering, do you need the compressible potential flow element to compute the adjoint of the incompressible one?

Reply:
Good point. During the implementation we realized that the incompressible_adjoint_potential_flow_element that Marc implemented is actually flexible and also admits the compressible element as primal. Therefore we will change the name to: adjoint_potential_flow_element